packages/grid/src/vaadin-grid-column.js

```diff
@@ -724,6 +737,22 @@ export const ColumnBaseMixin = (superClass) =>
 
       return this._defaultFooterRenderer;
     }
+
+    /**
+     * Updates part attribute on the cell depending on the state.
+     *
+     * @param {HTMLElement} cell
+     * @param {string} state
+     * @param {string | boolean | nul} value
+     * @param {?string} newPart
+     * @param {?string} oldPart
+     * @private
+     */
+    __updateCellPart(cell, state, value, newPart, oldPart) {
+      if (this._grid) {
+        this._grid._updateCellState(cell, state, value, newPart, oldPart);
+      }
+    }
```

Comment:
This is better than the duplicated grid existence checks previously, however this only seems to be necessary because we chose to store the update methods on the grid. We could consider moving them to a utility module, as the functions don't reference the grid instance.

Reply:
Good point, thanks. Will do.